Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:9-alpine
+
+RUN apk --no-cache add bash git
+# python build-base
+# remove python and build-base when grpc-prebuilt will be up again
+
+ADD . /opt/orbs
+
+WORKDIR /opt/orbs
+
+RUN npm install -g typescript@2.6.2
+
+RUN ./install.sh
+
+RUN find . -name package-lock.json -delete
+
+RUN ./build.sh
+
+RUN cd e2e && ./build.sh
```

Comment:
Should we add `e2e` as part of the main `build.sh`?

Reply:
We could, but at some point we're going to exclude it anyway, and probably have a different Dockerfile for production builds.

Reply:
Makes sense, yeah.